src/core/seed.js

```diff
@@ -4,23 +4,34 @@
 // Seed support
 //
 
+const {Secp256k1Pair, Ed25519Pair} = require('ripple-keypairs');
 const extend = require('extend');
 const utils = require('./utils');
+
 const sjcl = utils.sjcl;
 
 const Base = require('./base').Base;
 const UInt = require('./uint').UInt;
 const UInt160 = require('./uint160').UInt160;
-const KeyPair = require('./keypair').KeyPair;
 
 const Seed = extend(function() {
   this._curve = sjcl.ecc.curves.k256;
   this._value = NaN;
+  this._version = Base.VER_FAMILY_SEED;
 }, UInt);
 
 Seed.width = 16;
 Seed.prototype = extend({}, UInt.prototype);
 Seed.prototype.constructor = Seed;
+Seed.versions = [Base.VER_FAMILY_SEED, Base.VER_ED25519_SEED];
+
+// Backwards compatibility hack
+Secp256k1Pair.prototype.get_address =
```

Comment:
How much should we strive to maintain backwards compatibility?  

Candidates:  get_address /  pub_key_hex   ?